### PR TITLE
✨ styled-components #14

### PR DESCRIPTION
**Pull request by alex-wilmer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,12 +14,12 @@
...
     "babel-polyfill": "^6.23.0",
     "color": "^1.0.3",
-    "glamor": "^2.20.12",
     "react": "^15.5.4",
     "react-dom": "^15.5.4",
     "react-rotating-text": "^1.2.0",
     "react-router-dom": "^4.1.1",
-    "recompose": "^0.23.1"
+    "recompose": "^0.23.1",
+    "styled-components": "2.0.0-15"
```
Comment (tazsingh) [inline package.json]: Note if you're server side rendering, you'd also want to use https://github.com/styled-components/babel-plugin-styled-components

**Comment by alex-wilmer** [comment]
I'm not sure github pages can support server side rendering since.. well.. it's just some github server somewhere that we can't write special react code for. If SSR is worth it that much then we should consider hosting ourselves (digital ocean, etc). 

I'm not sure that SSR is worth the effort in this case. Maybe if the site starts picking up a lot of traffic we could move to a self hosted solution? Or maybe there is some github SSR magic I'm unaware of?

**Comment by tazsingh** [comment]
I mean more that the components themselves should be built statically and then the files are just hosted on Github. Not that we have a server somewhere that's statically rendering the files on demand.

**Comment by alex-wilmer** [comment]
Interesting. Definitely something to investigate! Do you have an example project? (Was this in your presentation that I missed? 😝 )

**Comment by tazsingh** [comment]
Wasn't in any presentation. Just something that I've done when pushing to CDNs and such. Part of general "Webpack-as-a-backend" architecture.

If I have some time after GraphQL Europe, I can contribute this architecture to those codebase. But until then, it's just optimizations really.

**Pull request merged**